### DISCUSSION TITLE
ui.upload with multiple file formats ".props('accept=.jpeg,.jpg,.png')" does not work

### DISCUSSION THREAD

**Discussion by miccs0** [zauberzeug/nicegui#3128] [Q&A] [answered]
### Question

Hello, I have this code:
```py
ui.upload(label="Photo", auto_upload=True, max_files=1).props('accept=.jpeg')
```
Which works, but I would like to accept multiple image files, like .png, .jpg, etc. 
When trying to do something like:
`.props('accept=.jpeg,.jpg,.png')`

It ignores the restriction.
THANKS!

**Comment by falkoschindler** [comment, marked as answer]
Thanks for pointing this out, @miccs0!
The reason is that commas are not (yet) a valid part of a prop string. Therefore you need to wrap it in double quotes:
```py
.props('accept=".jpeg,.jpg,.png"')
```
But to make it more convenient and consistent, I created PR #3130.

**Reply by miccs0** [reply]
Thanks! This worked perfectly.
